build.gradle

```
@@ -67,7 +67,30 @@ intellij {
 patchPluginXml {
     sinceBuild = "233.00"
     untilBuild = ""
-    pluginDescription = extractPluginDescription()
+    pluginDescription = """
+        <p>
+        Support for developing Jenkins plugins and Jenkins core.
+        Essential if you are a developer making a change to Jenkins or a Jenkins plugin.
```

```suggestion
Essential support for developing Jenkins plugins and core, crucial for developers working on related changes.
```

Not a fan of the repetition/plugins on the first line is first but second on the second line.

---

What about this?

```suggestion
        Essential support for developing Jenkins plugins and Jenkins itself.
```